provision/aws/route53/route53_create_failover_lambda.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -e -o pipefail
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/route53_common.sh
+
+if [ -z "${HEALTH_CHECK_ID}" ]; then
+  echo "HEALTH_CHECK_ID variable must be set"
+  exit 1
+fi
+
+ALARM_NAME=${HEALTH_CHECK_ID}
+TOPIC_NAME=${HEALTH_CHECK_ID}
+FUNCTION_NAME=${HEALTH_CHECK_ID}
+ROUTE53_REGION="us-east-1"
+
+export AWS_REGION=${ROUTE53_REGION}
+
+TOPIC_ARN=$(aws sns create-topic --name ${TOPIC_NAME} \
+  --query "TopicArn" \
+  --tags "Key=HealthCheckId,Value=${HEALTH_CHECK_ID}" \
+  --output text
+)
+
+# Create CloudWatch alarm
+aws cloudwatch put-metric-alarm \
+  --alarm-actions ${TOPIC_ARN} \
+  --actions-enabled \
+  --alarm-name ${ALARM_NAME} \
+  --dimensions "Name=HealthCheckId,Value=${HEALTH_CHECK_ID}" \
+  --comparison-operator LessThanThreshold \
+  --evaluation-periods 1 \
+  --metric-name HealthCheckStatus \
+  --namespace AWS/Route53 \
+  --period 60 \
+  --statistic Minimum \
+  --threshold 1.0 \
+
+# Create the Role used to execute the Lambda
+ROLE_ARN=$(aws iam create-role \
+  --role-name ${FUNCTION_NAME} \
+  --assume-role-policy-document \
+  '{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+          "Service": "lambda.amazonaws.com"
+        },
+        "Action": "sts:AssumeRole"
+      }
+    ]
+  }' \
+  --query 'Role.Arn' \
+  --output text
+)
+
+# Create a policy with the permissions required by lambda.py
+POLICY_ARN=$(aws iam create-policy \
+  --policy-name ${FUNCTION_NAME} \
+  --policy-document \
+  '{
+      "Version": "2012-10-17",
```

Comment:
why is the version a date here ?

Reply:
I believe this is just a quirk of AWS, as this value was the default when creating policies and is found in all examples. We use the same value when creating policies in `aurora_create_global_db.sh`.